jdaviz/app.py

```diff
@@ -1068,6 +1071,12 @@ def _get_range_subset_bounds(self, subset_state,
                      "subset_state": subset_state}]
         return spec_region
 
+    def _get_multi_mask_subset_definition(self, subset_state):
+        return [{"name": subset_state.__class__.__name__,
```

Comment:
Ah, you're explicitly wrapping the dict in a list here. What was the reason for this?

Reply:
The reason for this is that MultiMask could be one region of many in a composite subset. To return all that information, we use a list of dictionaries which contain information on those subregions.

Reply:
Ahh, I missed (or forgot) that `get_subsets()` always returns a list of dictionaries now, even the subset is a single region. As long as it's consistent, carry on nothing to see here 😄